client/src/app/LandingPage.css

```diff
@@ -0,0 +1,49 @@
+.sec1-container{
+    display: flex;
+    width: 100%;
+    height: 800px;
+    justify-content: center;
+    align-items: center;
+}
+
+.sec1-box{
+    display: flex;
+    width: 90%;
+    height: 700px;
+    background-color: white;
+    border: 10px solid #E0E4F2;
+    box-shadow: #6d6d6d 0px 0px 10px;
+    border-radius: 20px; 
+    align-items: center;
+    justify-content: center;
+}
+
+.sec1-logo-box{
+    display: grid;
+    width: 400px;
+    height: 500px;
+    position: absolute;
```

Comment:
I try not to use absolute positioning. I only use it if I need to take a component out of the flow like a floating component. Try not to rely on absolute for positioning components, it's a pain in the ass to debug if we style components like this

Reply:
fixed